cfme/middleware/provider/__init__.py

```diff
@@ -401,9 +404,14 @@ def add_datasource(self, ds_type, ds_name, jndi_name, driver_name,
         """
         self.load_details(refresh=True)
         datasources_btn("Add Datasource", invokes_alert=True)
+        if version.current_version() >= '5.8':
```

Comment:
REQUIRED: please don't use current_version. here you can use self.appliance.version() instead